bin/MadGraph5_aMCatNLO/patches/0021-fix-singularity-ld-library-path-with-scramrtdel.patch

```diff
@@ -152,7 +152,7 @@
 +export TMPDIR=${TMPDIR:-${_CONDOR_SCRATCH_DIR:-/tmp}}
 +
  tarCounter=0
- while [[ (-f MadLoop5_resources.tar.gz) && (! -f MadLoop5_resources/HelConfigs.dat) && ($tarCounter < 10) ]]; do
+ while [[ (-f MadLoop5_resources.tar.gz) && (! -f MadLoop5_resources/HelConfigs.dat) && ($tarCounter < 10
      if [[ $tarCounter > 0 ]]; then
```

Comment:
can you double check this? `while` isn't closed, a bit weird

Reply:
Ah yes. You are right. I don't know how it comes;; Let me check whether there is anything else like this.

Reply:
The others look fine.

Reply:
so we should add `) ]]; do` back in?

Reply:
Yes.